tests/testthat/test-bbr.R

```diff
@@ -96,3 +96,10 @@ test_that("bbi_init passes .bbi_args [BBR-BBR-008]", {
   expect_equal(bbi_yaml$threads, 36)
   expect_true(bbi_yaml$no_shk_file)
 })
+
+test_that("bbi_help captures and relays bbi output [BBR-BBR-009]", {
+  withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
+    res <- paste(capture.output(bbi_help()), collapse = "\n")
+  })
+  expect_true(stringr::str_detect(res, "bbi cli version"))
+})
```

Comment:
It'd be good to test `bbi init` too, but that command doesn't look very chatty.  I wasn't able to come up with a way to invoke `bbi_init()` that would lead to output from `bbi init` (versus a quiet success or a failure caught by `bbi_init()` _before_ calling `bbi init`).

Reply:
makes sense. I can't think of any messages that `bbi init` _should_ relay in any obvious scenarios.